### PR TITLE
Explain Q and F

### PR DESCRIPTION
**Pull request by timmoreton**
### Description

Add a comment to explain `MinQuorumSize` and `F`.

### Backwards compatibility

Yes

### CODE REVIEW

File: consensus/istanbul/validator/default.go [hunk 1]
```diff
@@ -247,6 +247,27 @@ func (valSet *defaultSet) Copy() istanbul.ValidatorSet {
 	return newValSet
 }
 
+// We include the optimization described at https://arxiv.org/pdf/1901.07160.pdf as “PM-6” and
+// discussed in Lemma 22. For values of N=3F for integer F=1,2,3,.. we can tolerate a quorum
```
Comment (nategraf) [inline consensus/istanbul/validator/default.go]: Should this be `N>=3F`?
```suggestion
// discussed in Lemma 22. For values of N>=3F for integer F=1,2,3,.. we can tolerate a quorum
```

File: consensus/istanbul/validator/default.go [hunk 2]
```diff
@@ -247,6 +247,27 @@ func (valSet *defaultSet) Copy() istanbul.ValidatorSet {
...
+// size one smaller than anticipated by Q = N - F. The intersection of any two sets of Q
+// nodes of N=3F must contain an honest validator.
```
Comment (nategraf) [inline consensus/istanbul/validator/default.go]: Same comment as above
```suggestion
// nodes of N>=3F must contain an honest validator.
```

File: consensus/istanbul/validator/default.go (hunk 1 above)
Comment (timmoreton) [reply consensus/istanbul/validator/default.go]: I think no, just specifically those `N` that are exact multiples of `3F` ?

Comment (nategraf) [reply consensus/istanbul/validator/default.go]: Ah I see now. You are discussing this specific case in this paragraph.

**Pull request merged**